### DISCUSSION TITLE
Twitter URL input shows '@' in the placeholder

### DISCUSSION THREAD

**Discussion by dinkydani** [paircodedev/sandbox#11] [Bug Report] [answered]
The prefix part suggests this is a URL so the handle should be without the @ symbol

<img width="418" alt="image" src="https://user-images.githubusercontent.com/3687223/120090520-d7a0d000-c135-11eb-9b83-1ef4d6424cf0.png">

**Comment by Necmttn** [comment, marked as answer]
Fixed 🎉
